docs/user_guide/write_api/push_job.md

```diff
@@ -53,6 +61,9 @@ The user may choose to specify the following configs:
 - `extended.schema.validity.check.enabled`: Whether to perform extended schema validation on the input (equivalent to
   the `STRICT` mode in avro-util's [SchemaParseConfiguration](https://github.com/linkedin/avro-util/blob/master/helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/SchemaParseConfiguration.java)). 
   If set to `false`, it becomes equivalent to avro-util's `LOOSE` mode. Default: `true`
+- `targeted.region.push.enabled`: Whether to perform targeted region push. Default: `false`
+- `targeted.region.push.list`: Optionally specify a list of target region(s) to push data into. See full details at 
```

Comment:
Mention this config takes effect only target push is enabled, otherwise not clear what `optionally` means here

Reply:
Updated.